### PR TITLE
fix onItem select incase of hover suggestions

### PR DESCRIPTION
**Pull request by santosh1994**
@rahulcs @floydpraveen Please review.
/cc @amarts

### CODE REVIEW

File: unbxdAutosuggest.js [hunk 1]
```diff
@@ -1071,23 +1073,23 @@ var unbxdAutoSuggestFunction = function($,Handlebars,undefined){
 		,getfilteredPopularProducts: function() {
 			var self = this;
 			var url = "http://search.unbxdapi.com/" + this.options.APIKey + "/" 
-				+ this.options.siteName + "/search?q=" + encodeURIComponent(this.params.q) + '&rows=' + this.options.popularProducts.count;
+				+ this.options.siteName + "/search?q=" + encodeURIComponent(this.params.q) + '&rows=' + this.options.popularProducts.count +'&indent=off&facet=off';
```
Comment (rahulcs) [inline unbxdAutosuggest.js]: @santosh1994 how is `&indent=off&facet=off` part of this PR?

Comment (santosh1994) [inline unbxdAutosuggest.js]: @rahulcs This should have been sent in other commit. Will update the PR

**Comment by santosh1994** [comment]
@rahulcs Please review. Made changes

**Comment by amarts** [comment]
@rahulcs on you to review.

**Comment by santosh1994** [comment]
this has been handled in #57

**Pull request closed without merging**